template_content/playground/abi/application.py

```diff
@@ -0,0 +1,16 @@
+import beaker
```

Comment:
In one of the slack messages from memory we settled on contract.py - not suggesting that change should be made in this pull request, but wanted to check if we are still happy with that?

Reply:
I'm ok with it but I also don't have a strong feeling either way

Reply:
It should be a meaningful name, e.g. `helloworld.py` so that it's easier to read in code that uses the contract module